src/__tests__/RenderHomePage.test.js

```diff
@@ -20,16 +15,9 @@ describe('<RenderHomePage /> test suite', () => {
     const { getByText } = render(
       <Provider store={store}>
         <Router>
-          <RenderHomePage
-            userInfo={{ name: 'Sara' }}
-            authService={authService}
-          />
+          <RenderHomePage authService={authService} />
         </Router>
       </Provider>
     );
-    const button = getByText(/Sign Out/i);
```

Comment:
I agree that this may not be the best test for use case. However we do need to replace this test with one that works for our app.